### PR TITLE
remove repeat failure command option and increase timeouts in etcd failure e2e

### PR DESCRIPTION
**Pull request by mikedanese**
#10144

@quinton-hoole @davidopp

### CODE REVIEW

File: test/e2e/etcd_failure.go [hunk 1]
```diff
@@ -121,7 +108,7 @@ func checkExistingRCRecovers(f Framework) {
 	rcSelector := labels.Set{"name": "baz"}.AsSelector()
 
 	By("deleting pods from existing replication controller")
-	expectNoError(wait.Poll(time.Millisecond*500, time.Second*30, func() (bool, error) {
+	expectNoError(wait.Poll(time.Second*5, time.Second*60, func() (bool, error) {
```
Comment (quinton-hoole) [inline test/e2e/etcd_failure.go]: You could probably leave the retry period at 500ms.  I don't feel terribly strongly about that though.

**Comment by davidopp** [comment]
LGTM

**Comment by quinton-hoole** [comment]
This only:

1. reduces the number of SIGKILL's sent to etcd from 20 to 1
2. increases the timeout allowed for etcd recovery from 30 to 60 seconds.

It's a safe, test-only change, and gives us a very useful, and hopefully now stable e2e test.

**Comment by quinton-hoole** [comment]
Fixes #10144

**Pull request merged**